app/app.ino

```diff
@@ -346,11 +350,18 @@ void setup()
 
     initSensor();
     // setup iot hub client which will diliver your message
+    
+    /*
+    * Break changes in version 1.0.34 the AzureIoTHub Library removed AzureIoTClient class.
+    * So we remove the code below to avoid compile error.
+    */
+
+    /*
     beginIoThubClient();
```

Comment:
1. Client sync time to calculate the SAS token. If here remove this part of code, where to sync time for IoT hub client? Could you please double check that?

2. Separate the code comment and doc comment even if the code comment need to be kept.